### ISSUE TITLE
Add technique explaining WCAG 2.4.6 passes for all publication headings

### ISSUE THREAD

**Issue by mattgarrish** [mattgarrish/epub-revision#1810]
2.4.6 is the AA success criterion that says "Headings and labels describe topic or purpose".

While this makes sense for labels, and for books with topic-based chapters, how a novel chapter heading describes its topic or purpose isn't immediately clear in a lot of cases. It's questionable if a chapter number alone describes either a topic or purpose (short of an argument that the Xth chapter's purpose is to be in position X).

I finally got around to opening an issue about this in the [WCAG tracker](https://github.com/w3c/wcag3/issues/170), and it seems the intent of the success criterion doesn't exactly match its wording, but it's also complicated to change the SC text now.

The general response I've gotten, though, is that the presence of a heading establishes the context of its chapter (if not a specific topic or purpose) and that's enough to satisfy the SC, so it's never actually an issue for publications.

Should we document this in the techniques document, though, so we have it as a reference?

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-09-23-epub-a11y#section2) on 2021-09-23

List of resolutions:

- ***[Resolution No. 2](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-09-23-epub-a11y#resolution2):  Add explainer for WCAG 2.1 SC 2.4.6 in EPUB a11y techniques***

<details><summary><i>View the transcript</i></summary>

### 2. Add technique explaining WCAG 2.4.6 passes for all publication headings 

_See github issue [#1810](https://github.com/w3c/epub-specs/issues/1810)._




**Avneesh Singh:** Next one is also from mgarrish  

**Matt Garrish:** A question that's sat in the back of my head  
… all headings must describe the topic of their section  
… which doesn't quite make sense for things like novels  
… what's their purpose and role  
… I opened the issue and was confirmed that they SCs don't quite cover all cases, it just needs to be a descriptive identifier for the content that follows  
… it's kind of problematic, also lumped with labels  
… it's too big a change for WCAG 2  
… good for WCAG 3 where we can discuss further  
… should we document this  
… how does WCAG apply to EPUB  
… it's a known issue with the wording  
… not to be concerned with the heading missing a topic  
… fits in with our techniques MO  

**George Kerscher:** Even numbers are fine  

**Matt Garrish:** That's what threw me off, there's context in numbers, but it's not descriptive of the content  
… it's going after a different problem, headings that don't match the content  
… do we want to document it  

**Avneesh Singh:** I think documenting it is good  
… one of the purposes of techniques is to explain how WCAG applies  

> **Proposed resolution: Add explainer for WCAG 2.1 SC 2.4.6 in EPUB a11y techniques** *(Avneesh Singh)*

> *Matt Garrish:* +1

> *Wendy Reid:* +1

> *Gregorio Pellegrino:* +1

> *Bill Kasdorf:* +1

> *Murata Makoto:* +1

**George Kerscher:** +1  

> ***Resolution w3c/wcag#2: Add explainer for WCAG 2.1 SC 2.4.6 in EPUB a11y techniques***

</details>
